rfcs/0003-storage.md

```diff
@@ -1,46 +1,112 @@
-# Storage
+# 1. Storage
```

Comment:
No need for the `1.`, `2.` in the headers. Not too useful in the RFC and when migrated to the book, the HTML generator will add them automatically.

Reply:
Well I would like to reference a specific section in an issue to make it clear what that issue is referring to. Any suggestions for that?

Reply:
What does it mean?